src/AesDecryptingStream.php

```diff
@@ -117,4 +127,4 @@ private function decryptBlock(int $length): string
 
         return $plaintext;
```

Comment:
`openssl_decrypt` has the potential to return false. This could cause a PHP error as the return type hint is `string` - does this need to be `return $plaintext ?: '';` maybe?

Reply:
Agree, though maybe decryption failures (which I think would usually be due to supplying the wrong key, IV, or method) should throw instead. Let's leave that for another PR so that the fix for the bug you identified can be released as a patch version. (I would want to bump from `0.3.x` to `0.4.0` with a behavior change like what you propose.)